sound/soc/sof/control.c

```diff
@@ -71,7 +71,7 @@ int snd_sof_volume_put(struct snd_kcontrol *kcontrol,
 
```

Comment:
@plbossart @lgirdwood Also, it might worth having a look at mailbox API (see drivers/mailbox). I think SOF IPC layer is just another version of a mailbox controller.

Reply:
@dbaluta @plbossart there is a lot of design duplication in rpmsg and mailbox, but they are targetted at different levels in the stack. I think rpmsg is the best fit for SOF once we can upgrade to IPC2.0 since it should fit nicely above the IPC layer (mailbox would fit below - but we dont share below IPC).

Reply:
@plbossart @lgirdwood I was thinking the same when I started working on this implementation. Using rpmsg for the common IPC layer that is shared between multiple clients seems like the way to go. I can look into it after I finish with the current changes to remove the MFD API's in this PR